src/modules/cmdpal/Microsoft.CmdPal.UI/ExtViews/ListPage.xaml

```diff
@@ -33,29 +33,12 @@
             Spacing="8" />
 
         <DataTemplate x:Key="TagTemplate" x:DataType="viewmodels:TagViewModel">
-            <StackPanel
-                Padding="4,2,4,2"
-                VerticalAlignment="Center"
-                Background="{x:Bind help:OptionalColorBrushCacheProvider.Convert(Background, x:False), Mode=OneWay}"
-                BorderThickness="1"
-                CornerRadius="4"
-                Orientation="Horizontal">
-                <cpcontrols:IconBox
-                    x:Name="IconBorder"
-                    Width="12"
-                    Height="12"
-                    Margin="0,0,4,0"
-                    Foreground="{x:Bind help:OptionalColorBrushCacheProvider.Convert(Foreground, x:True), Mode=OneWay}"
-                    SourceKey="{x:Bind Icon, Mode=OneWay}"
-                    SourceRequested="{x:Bind help:IconCacheProvider.SourceRequested}"
```

Comment:
@zadjii-msft @michael-hawker Not sure what the best approach is for moving this into the control. Can't use binding to a staticresource because of AOT, right? x:Bind wouldn't work on a Templated control either. 

Should we have a dependency property in `Tag` that allows you to set this? Or should we set the eventhandler in the Tag initialization?